ec2/spark_ec2.py

```diff
@@ -706,9 +697,7 @@ def wait_for_cluster_state(conn, opts, cluster_instances, cluster_state):
     sys.stdout.flush()
 
     start_time = datetime.now()
-
     num_attempts = 0
-    conn = ec2.connect_to_region(opts.region)
```

Comment:
I noticed we were overwriting `conn` here so I removed this line, though it's unrelated to this patch.